tests/integration/routing/07c-community-delete-list.yml

```diff
@@ -0,0 +1,64 @@
+---
+message: |
+  Use this topology to test 'set.community.delete' route-map option.
+
+plugin: [ bgp.policy ]
+module: [ bgp, routing ]
+
+groups:
+  probes:
+    device: frr
+    provider: clab
+    members: [ probe, x1 ]
+
+routing.community.c_42: '65101:42'
+
+nodes:
+  dut:
+    bgp.as: 65000
+    routing.policy.set_comm:
+      set.community:
+        delete_list: c_42
```

Comment:
Should test deleting different types of communities - standard, extended, large

Reply:
ok, I missed that - wording could distinguish more clearly between what community lists "are" and how Netlab implements them

Reply:
However, given that 'delete' supports all community types, how would you handle a delete of an extended community on FRR?

Reply:
https://github.com/ipspace/netlab/blob/dev/netsim/modules/routing.py#L484 (added in this PR)

Reply:
Documented in https://github.com/ipspace/netlab/blob/dev/docs/module/routing.md?plain=1#L157 (also part of this PR)